gateway-home/modules/ROOT/pages/index.adoc

```diff
@@ -1,53 +1,85 @@
+Mule runtime engine includes an embedded Mule Gateway. Mule Gateway enables you to add a dedicated orchestration layer on top of your backend APIs and services to help you separate orchestration from implementation. You can leverage the governance capabilities of API Manager to apply, among other capabilities, throttling, security, caching, and logging to your APIs.
 
-Overall, Mule Gateway allows you to add a dedicated orchestration layer on top of your backend APIs and services to help you separate orchestration from implementation concerns. You can leverage the governance capabilities of API Manager to apply, among other capabilities, throttling, security, caching, and logging to your APIs.
+Using Mule Gateway, any user can, for example, apply a basic authentication policy on top of a Mule application, enrich an incoming/outgoing message, or add any other complex capability to an API without having to write any code.
 
 If you want to apply policies and generate analytics info, you must use either:
 
 * An API defined in a Mule application.
 * A Mule application that proxies your existing backend implementation.
 
-For more information, see xref:mule-gateway::mule-gateway-capabilities-mule4.adoc[Mule Gateway Capabilities].
+For more information, see xref:mule-gateway::mule-gateway-capabilities-mule4.adoc[].
+
+== Flex Gateway vs. Mule Gateway
+
+While Flex Gateway is a high-performance Envoy-based gateway with a small footprint, MuleSoft also provides a simple Mule Gateway for Mule applications.
+
+Mule runtime engine, based on Java Spring, includes an embedded Mule Gateway. By using this gateway, you can, for example:
+
+* Apply a basic authentication policy on top of a Mule application.
+* Enrich an incoming or outgoing message.
+* Add any other complex capability to an API without having to write any code.
+
+Mule Gateway protects a single Mule API. The key advantage is that it's easy for Mule app developers to provide basic endpoint protection.
+
+You can also configure Mule in Anypoint Runtime Manager as a CloudHub Proxy application, protecting multiple backends. However, in this case the gateway cannot be easily scaled to support high-availability and high-performance applications. Further, Mule consumes more computes or vCores than the smaller-footprint Flex Gateway.
+
+Building custom policies on Mule Gateway is similar to building an application with Java using the Mule DSL. Building a custom policy in Flex Gateway is based on Envoy-provided Rust WASM SDKs. A Mule Gateway policy cannot be reused in Flex Gateway and vice versa, because the underlying architectures are fundamentally different.
+
+MuleSoft recommends you choose Flex Gateway for high-availability and high-performance applications.
+
+[%header%autowidth.spread,cols="a,a,a"]
+|===
+| | Flex Gateway | Mule Gateway
+
+| *Summary*
+| Envoy-based API gateway to secure all APIs, Mule and non-Mule, running anywhere.
+| Java-based API gateway embedded into Mule.
```

Comment:
Mention only Mule?

Reply:
@hannanelson Sorry, I'm not sure I follow. Would you elaborate?

Reply:
One thing I look at with comparison tables is making the columns parallel. So, if you mention "all APIs" in the Flex column, mention "only Mule APIs" in the Mule Gateway column.

Reply:
Ah, got it. Fixed.